lib/YaST/DNSServer/MainPage.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles DNSServer Main Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::DNSServer::MainPage;
+use parent 'Installation::Navigation::NavigationBase';
```

Comment:
this was designed for the bottom bar of the installation with UI controls:
`Help`, `Abort`, `Back` and `Next`.
I don't see much use here, as most of the modules will have:
`Help, `Cancel`, `Apply Changes` and `OK`.
Could you please create something like: `YaST::Navigation::NavigationBase` with all those options to be used for future automation for YaST modules? wdyt?